lib/shoulda/matchers/active_record/have_db_column_matcher.rb

```diff
@@ -137,6 +138,13 @@ def description
 
         protected
 
+        OPTIONS = %i(precision limit default null scale primary)
```

Comment:
Would you mind putting this at the top of the class? That way it "pops" out.

Reply:
I moved but I've a question about the visibility of this constant. Shoud I use something to hide like `#private_constant`?

Reply:
Sure, that would make sense if you want to do that.